test/test_utils/WireMockHelper.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package utils
+package test_utils
```

Comment:
utils will have been fine here as this sits under the test folder structure

Reply:
There is utils package inside project which also need to be tested and should sit in utils package this is why I rename this one to avoid confusion